### PR TITLE
doc: delete some text

### PR DESCRIPTION
**Pull request by ThibaultFy**
Signed-off-by: ThibaultFy <50656860+ThibaultFy@users.noreply.github.com>

### CODE REVIEW

File: docs/source/contributing/backend/index.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Kubernetes pods
 ===============
 
 docker-registry
-    We use this service to store images built from user-provided :ref:`Algorithms<concept_algorithm>`.
+    to store images built from user-provided :ref:`Algorithms<concept_algorithm>`.
```
Comment (ThibaultFy) [inline docs/source/contributing/backend/index.rst:13]: Why did delete this ?

File: docs/source/contributing/backend/index.rst [hunk 2]
```diff
@@ -20,11 +20,11 @@ minio
     You should back up the data of this Pod.
 postgresql
     This is the database supporting the Backend.
-    You should back up the data of this Pod.
+    You should backcvds up the data of this Pod.
```
Comment (ThibaultFy) [inline docs/source/contributing/backend/index.rst:23]:
```suggestion
    You should back up the data of this Pod.
```

Comment (guilhem-barthes) [inline docs/source/contributing/backend/index.rst:23]: backcvds ?

**Review by ThibaultFy — COMMENTED** [review]
Thanks a lot

File: docs/source/contributing/backend/index.rst [hunk 3]
```diff
@@ -20,11 +20,11 @@ minio
...
 redis
     This is an organization-specific message broker to support `Celery`_ tasks.
 backend-events
-    This component will consume events from the Orchestrator.
+    This component will consu events from the Orchestrator.
```
Comment (guilhem-barthes) [inline docs/source/contributing/backend/index.rst:27]: consu ?

**Review by guilhem-barthes — CHANGES_REQUESTED** [review]

**Pull request closed without merging**